rbd-target-api.py

```diff
@@ -538,6 +545,97 @@ def _gateway(gateway_name=None):
             return jsonify(message="Gateway removed successfully"), 200
 
 
+def define_luns(gateway):
+    """
+    define the disks in the config to LIO
+    :param gateway: (object) gateway object - used for mapping
+    :return: None
+    """
+
+    local_gw = this_host()
+
+    # sort the disks dict keys, so the disks are registered in a specific
+    # sequence
+    disks = config.config['disks']
+    srtd_disks = sorted(disks)
+    pools = {disks[disk_key]['pool'] for disk_key in srtd_disks}
+
```

Comment:
Why did we have to do this by pool? Can we just loop over "disks" in gateway.conf and add them? Something like:

for disk in config.config['disks']:
    lun = LUN(logger, disk['pool'], disk['image'], rbd_image.size, disk['owner])
    # check for error like below
    lun.allocate()
    # check for error like below
   
gateway.manage('map')
# check error like below